docs/configuration.rst

```diff
@@ -153,6 +161,23 @@ Some more examples:
     have multiple components that share configuration like credentials and
     hostnames.
 
+``config('port', default='8000', parser=int, doc='the port you want this to listen on')``
+    You can provide a ``doc`` argument which will give users users who are trying to
+    configure your software a more helpful error message when they hit a configuration
+    error.
+
+    Example of error message without ``doc``::
+
+        everett.InvalidValueError: ValueError: invalid literal for int() with base 10:
+        'bar'; namespace=None key=foo requires a value parseable by int
+
+    Example of error message with ``doc``::
+
+        everett.InvalidValueError: ValueError: invalid literal for int() with base 10:
+        'bar'; namespace=None key=foo requires a value parseable by int
+        the port you want this to listen on
```

Comment:
This additional line would have come from the ``doc`` argument.

Mmm... looking at this, it makes me want to capitalize the first word and put a . at the end.

Do you think this looks helpful? Are there ways we could make this more helpful?